circuits/cpp/barretenberg/cpp/src/barretenberg/honk/pcs/shplonk/shplonk.test.cpp

```diff
@@ -65,7 +66,7 @@ TYPED_TEST(ShplonkTest, ShplonkSimple)
     auto verifier_transcript = VerifierTranscript<Fr>::init_empty(prover_transcript);
 
     // Execute the shplonk verifier functionality
-    const auto verifier_claim = Shplonk::reduce_verify(this->vk(), opening_claims, verifier_transcript);
+    const auto verifier_claim = ShplonkVerifier::reduce_verify(this->vk(), opening_claims, verifier_transcript);
```

Comment:
I would suggest renaming the `reduce_verify` functions to either just `verify` or a more descriptive name that is protocol-related

Reply:
I think reduce verify was originally chosen because its not really verifying anything, its just creating a batched commitment that will be used ultimately in verification. So `verify` might be misleading. The analog to the corresponding prover method would be `compute_partially_evaluated_batched_quotient_commitment`. Too long?

Reply:
very :) maybe it should be `reduce_verification` ?

Reply:
yeah or maybe even just `compute_batched_commitment`. It's not fully descriptive but it gives a sense of what's actually being done

Reply:
Changed Gemini and Shplonk verifier methods to `reduce_verification`